src/main/java/edu/iastate/ece/sd/sdmay2126/application/ApplicationOutput.java

```diff
@@ -0,0 +1,12 @@
+package edu.iastate.ece.sd.sdmay2126.application;
+
+/**
+ * Output from some particular KBase application execution. Will correspond to some {@link ApplicationParameters} for
+ * the same {@link Application}.
+ */
+public abstract class ApplicationOutput {
```

Comment:
Any reason this and AppParams are abstract classes vs an interface? Are you planning on adding some superclass methods in the future?

Reply:
Nope, I'll change them to interfaces.